althea_types/src/interop.rs

```diff
@@ -684,6 +686,8 @@ pub struct OperatorCheckinMessage {
     /// This is a user set bandwidth limit value, it will cap the users download
     /// and upload to the provided value of their choosing. Denoted in mbps
     pub user_bandwidth_limit: Option<usize>,
+    /// Details of both the Client and Relay bandwidth usage over the last single hour period
+    pub user_bandwidth_usage: Option<UsageTracker>,
```

Comment:
We do use the last saved field as what we check against on the ops side to send back the ops_last_seen_usage_hour, I'm hesitant to try removing that and start setting that value based on an ops side implementation of get_current_hour because then it's possible that we may come out of sync(as in if a router sends out new data right before the start of a new hour since unix epoch and all of a sudden we may miss an hour if ops thinks it has just received the new hour)